core/pythonAction/pythonaction.py

```diff
@@ -56,12 +64,18 @@ def run():
     if not isinstance(value, dict):
         flask.abort(403)
 
+    if flask.g is None:
+        # no code to execute
+        response = flask.jsonify({"error": "No code to execute (init failed). See logs for details." })
```

Comment:
this should never happen btw (ie if you're called when init failed, there's an internal system error).